src/Symfony/Component/Validator/Resources/translations/validators.es.xlf

```diff
@@ -374,6 +374,14 @@
                 <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
                 <target>El número de elementos en esta colección debería ser múltiplo de {{ compared_value }}.</target>
             </trans-unit>
+            <trans-unit id="97">
+                <source>This value should satisfy at least one of the following constraints:</source>
+                <target>Este valor debería satisfacer al menos una de las siguientes restricciones:</target>
```

Comment:
@javiereguiluz Once again, I put my Spanish to the test.

Reply:
👍 The proposed translation is perfect! Thanks.